### ISSUE TITLE
Intermittent UI test failure - < SearchTest.noRecentlyVisitedSearchGroupInPrivateBrowsingTest>

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#24421]
### Firebase Test Run:
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/6481699293202036185/executions/bs.6598a70b52a38ef0/testcases/1/test-cases)
[Firebase link](https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/6481699293202036185/executions/bs.cf358555ecf880c8/testcases/1/test-cases)

### Stacktrace:
[Logcat](https://github.com/mozilla-mobile/fenix/files/8334514/verifyContextOpenImageNewTab.txt)
[Logcat](https://github.com/mozilla-mobile/fenix/files/8334515/verifyContextOpenImageNewTab2.txt)

**Comment by sv-ohorvath** [comment]
A11y issue causing the failure was fixed, can be reopened if it fails again.
